components/global_onboarding_list/onboarding_checklist.tsx

```diff
@@ -196,6 +196,11 @@ const TaskList = (): JSX.Element => {
             id='onboarding_checklist.task_system_console'
             defaultMessage='Visit the System Console to configure your workspace'
         />,
+        <FormattedMessage
+            key={7}
+            id='onboarding_checklist.task_start_trial'
+            defaultMessage='Learn more about Enterprise-level high-security features'
+        />,
```

Comment:
We should only show is they are not in cloud, haven't done a trial before and don't have a license (which most likely hasn't happened early in the onboarding but we should be consistent with the same checks we currently do to show this in the main menu)